### PR TITLE
add aurora tools to mth5

### PR DESCRIPTION
**Pull request by kujaku11**
Add some aurora tools that have a broader application

- [x] add `mth5.tables.channel_summary.to_run_summary()`
  - [x] add `mth5.MTH5.run_summary` convenience method

### CODE REVIEW

File: .github/workflows/tests.yml
Comment (kkappler) [inline .github/workflows/tests.yml:1]: Should the mt_metadata branch be changed back to `main` before merging ?

File: mth5/data/make_mth5_from_asc.py
Comment (kkappler) [inline mth5/data/make_mth5_from_asc.py:1]: might be good to meld this against `patches` branch before merge

**Review by kkappler — CHANGES_REQUESTED** [review]
Nice additions.  Run_summary is going to be very useful in future!

This looks ready to merge after verifying check tests.yaml is supposed to be on `anna` branch.

Minor Comments:
- Most changes shown in github were just black re-formatting.  It would be good to sync our precommit.yaml files so these changes are less common in future.
- I added some minor changes (improvements) to `make_mth5_from_ascii.py` on `patches` branch. Perhaps these could be be melded in?  But if not, I'll meld this with patches so no problem.  I guess, whichever hits main first will be the reference.
- I tested run_summary in a separate application and it seems to behave as expected.  Yay!

**Review by kujaku11 — COMMENTED** [review]
Changes are good and pass tests

**Pull request merged**
